src/test/java/mock/jvb/MockVideobridge.java

```diff
@@ -207,7 +200,11 @@ public IQ handleIQRequest(IQ iqRequest)
 
     public int getEndpointCount()
     {
+        System.out.println("Bridge has " + bridge.getConferences().size() + " conferences");
```

Comment:
why those prints here? it looks like some debugging left by mistake

Reply:
Oops, you're right. Thanks.